s30161-pj-NikodemCyrzan/zajecia1/instalacja-debiana-linux.md

```diff
@@ -0,0 +1,523 @@
+-   Separate /home, /var, and /tmp partitions - umieszcza /home, /var i /tmp na osobnych partycjach, dzięki czemu dane użytkowników, logi serwerów i dane tymczasowe mogą być bezpieczniej przechowywane.
+
+> Mój wybór
+>
+> `All files in one partition`, ponieważ nie umiem jeszcze używać systemów z wydzielonymi partycjami dla poszczególnych plików i nie wiem jeszcze do końca z czym to się wiąże.
+
+### Różnica między partycją primary, a logical
+
+primary - partycja przeznaczona do przechowywania systemu operacyjnego. Jest ona zapisana w konkretny sposób, dzięki czemu system może być prosto załadowany. Można na niej przechowywać również różne dane.
+
+logical - partycja przeznaczona tylko do przechowywania danych. Gdy tworzy się partycję logiczną, to tworzona jest również partycja extended (rozszerzona), która przechowywuje informacje o wszystkich partycjach logicznych na dysku.
+
+### Formaty systemów plików:
+
+-   Ext2 file system - szybki system plików bez księgowania. Księgowanie to odnotowywanie aktualnie trwającej operacji, dzięki czemu w razie przerwania pracy systemu na przykład z powodu braku prądku, system plików nie zostaie uszkodzony. Z powodu braku tej funkcji system narażony jest na awarie.
+-   Ext3 jurnaling file system - system plików podobny do Ext2, ale z księgowaniem. Z powodu księgowania jest wolniejszy niż Ext2. Jest wstecznie kompatybilny z Ext2.
+-   Ext4 jurnaling file system - system plików który jest szybszy od Ext2 pomimo posiadania księgowania. Nie jest wstecznie kompatybilny.
+-   btrfs jurnaling file system - system plików posiadający księgowanie i działający jak system kontroli wersji.
+-   JFS jurnaling file system - 64-bitowy system plików posiadający księgowanie. Był to jeden z pierwszych systemów z księgowaniem, ale współcześnie nie ma potrzeby używania go, ponieważ nowsze systemy są wydajniejsze od niego.
+-   XFS jurnaling file system - 64-bitowy system plików posiadający księgowanie. Jest bardzo wydajny w przypadku zapisu i odczytu wielkich plików. Umożliwia również równoległą pracę na plikach, dzięki czemu jednocześnie może być prowadzony zapis i odczyt.
+-   FAT16 file system - system plików posługujący się 16-bitowym adresowaniem, które ograniczało pojemność dysków do 32 MB.
+-   FAT32 file system - system plików posługujący się 32-bitowym adresowaniem wykorzystywany często w dyskach przenośnych jak Pendrive, czy dyskach zewnętrznych.
+-   swap area - dodatkowa przestrzeń na dysku służąca jako zapasowa pamięć RAM. Przestrzeń swap powinna być 2 razy większa niż pamięć RAM, jeśli jest jej mniej niż 2GB. W przeciwnym wypadku powinno być jej tyle co pamięć RAM + 2 GB.
+-   physical volume for encryption - partycja z przeznaczeniem do szyfrowania danych.
+-   physical volume for RAID - partycja z przeznaczeniem do tworzenia macierzy RAID, umożliwiających zwiększanie wydajności i niezawodoności dysków przechowujących dane.
```

Comment:
Partycja czy RAID nie zwiększa wydajności i niezawodności dysków.
Cała macierz jako taka daje taki efekt ale każdy dysk z osobna ma taką żywotność jaką ma i żadne rozwiązania jej nie zwiększą.
Trzeba to przeredagować.